lib/Page.js

```diff
@@ -290,6 +292,35 @@ class Page extends EventEmitter {
   }
 
   /**
+   * @return {!Promise<!Object>}
+   */
+  async getMetrics() {
+    const response = await this._client.send('Performance.getMetrics');
+    return this._buildMetricsObject(response.metrics);
+  }
+
+  /**
+   * @param {*} event
+   */
+  _emitMetrics(event) {
+    this.emit(Page.Events.Metrics, {
+      title: event.title,
+      metrics: this._buildMetricsObject(event.metrics)
+    });
+  }
+
+  /**
+   * @param {?Array<!{name: string, value: number}>} metrics
+   * @return {!Object}
+   */
+  _buildMetricsObject(metrics) {
+    const result = {};
+    for (const metric of metrics || [])
+      result[metric.name] = metric.value;
```

Comment:
since we don't want to expose certain numbers, can we white-list properties that we copy (instead of copying everything)?

Reply:
Done